objectivec/Tests/CocoaPods/run_tests.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+#
+# Helper to run the pods tests.
+
+set -eu
+
+readonly ScriptDir=$(dirname "$(echo $0 | sed -e "s,^\([^/]\),$(pwd)/\1,")")
+
+printUsage() {
+  NAME=$(basename "${0}")
+  cat << EOF
+usage: ${NAME} [OPTIONS]
+
+This script does the common build steps needed.
```

Comment:
Should this be more specific? "Runs a suite of build tests with CocoaPods to validate the Protobuf podspec", or something?